app/components/dropdown-search.js

```diff
@@ -0,0 +1,134 @@
+import Ember from "ember";
+import { singularize, pluralize } from "ember-inflector";
+import _ from "lodash";
+const { getOwner } = Ember;
+
+export default Ember.Component.extend({
+  selected: [],
+  previousValue: "",
+  store: Ember.inject.service(),
+  subformDetailService: Ember.inject.service(),
+  resourceType: Ember.computed.alias("packageDetails"),
+  fixedDropdownArr: ["frequency", "voltage", "compTestStatus", "testStatus"],
+  selectedData: Ember.computed.alias("selectedValue"),
```

Comment:
Why do we need to use an alias here? I think we can directly use `selectedValue`